pkgs/applications/audio/squeezelite/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchgit, alsaLib, flac, libmad, libvorbis, faad2, mpg123 }:
+
+stdenv.mkDerivation {
+  name = "squeezelite-git-2016-05-27";
+
+  src = fetchgit {
```

Comment:
Does this use git submodules? If not, you can use `fetchFromGitHub`, which expands an arbitrary revision to a url to a autogenerated tarball.